stream-chat-android-ui-components/src/main/res/values/styles.xml

```diff
@@ -506,8 +506,16 @@
         <item name="streamUiMediaAttachmentMoreCountTextSize">@dimen/stream_ui_message_media_attachment_more_count_text_size</item>
         <item name="streamUiMediaAttachmentMoreCountTextColor">@color/stream_ui_literal_white</item>
         <item name="streamUiMediaAttachmentMoreCountTextStyle">normal</item>
+    </style>
 
-        <item name="streamUiMediaAttachmentGiphyHeight">200dp</item>
+    <style name="StreamUi.MessageList.GiphyMediaAttachment" parent="StreamUi">
+        <item name="streamUiGiphyMediaAttachmentProgressIcon">@drawable/stream_ui_rotating_indeterminate_progress_gradient</item>
+        <item name="streamUiGiphyMediaAttachmentGiphyIcon">@drawable/stream_ui_giphy_label</item>
+        <item name="streamUiGiphyMediaAttachmentPlaceHolderIcon">@drawable/stream_ui_picture_placeholder</item>
+        <item name="streamUiGiphyMediaAttachmentImageBackgroundColor">@color/stream_ui_literal_transparent</item>
+        <item name="streamUiGiphyMediaAttachmentGiphyMaxHeight">200dp</item>
+        <item name="streamUiGiphyMediaAttachmentGiphyType">original</item>
```

Comment:
We have a slightly confusing situation here which is that our original size giphies come out very small and cropped out of the box. The most confusing part is that they are smaller than the lesser quality `fixedheight` ones. The originals also don't make the container follow the correct aspect ratio so they either get cropped or have extra spaces around.

If we allowed the originals to be resized as well as remove coercing max `widthToHeightRatio` we can have original quality Giphies splayed over the screen.  Now this creates a lot of jumping and a slightly confusing situation where wide Giphys cover all of the screen making it unclear who had sent it, but I presume we could mitigate that with padding.

However doing this once again re-introduces the bug we had before when Giphys that have text only wrap the text width and cut the Giphy off.

I've been looking at this for so long I'm unsure of the solution and would love a set of fresh eyes on this.

Maybe @bychkovdmitry could take a lot if he has the time?